### PR TITLE
Fix #5037 - In Mobile view the 'More' button behaviour is wrong when vie...

### PR DESCRIPTION
**Pull request by visualsayed**
work around until rails update

### CODE REVIEW

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -61,4 +61,40 @@ def jquery_include_tag
     buf << [ javascript_tag("$.fx.off = true;") ] if Rails.env.test?
     buf.join("\n").html_safe
   end
-end
+
+  # Replicates rails's [current_page?] method, so it respects the format appended to the url 
+  # We should remove this method when [rails] update it.
```
Comment (jhass) [inline app/helpers/application_helper.rb]: Please add a `# TODO: Revisit after Rails decision about https://github.com/rails/rails/pull/19378`

Comment (visualsayed) [inline app/helpers/application_helper.rb]: done, thanks

**Comment by jhass** [comment]
You might want to set your editor to strip trailing whitespace or respect our .editorconfig ;)

**Comment by svbergerem** [comment]
Here is some more information about editorconfig with a list of supported editors: http://editorconfig.org/

**Comment by visualsayed** [comment]
Done :-)
thanks

**Comment by jhass** [comment]
Mmmh, since this actually fixes a bug, a test that fails without this change would be nice.

**Comment by RichKavanagh** [comment]
@visualsayed have you had a chance to take another look at this mate?

**Comment by visualsayed** [comment]
@RichKavanagh I need to write some tests and it will be done. hopefully next Friday :-)

**Comment by denschub** [comment]
Closing due to excessive inactivity. Feel free to reopen if you want to continue working on this.

**Pull request closed without merging**